includes/cron/class-urlslab-serp-cron.php

```diff
@@ -90,22 +90,23 @@ protected function execute(): bool {
 		$types = implode( ',', $types );
 
 		global $wpdb;
-		$row = $wpdb->get_row(
+		$rows = $wpdb->get_row(
```

Comment:
there is other method to get results ... get_row is I think for one row only, get_results or something similar is for more rows

Reply:
If you want to use get_row, you need to specify random parameter y as parameter ... but it is not good, it will not finish all rows randomly ... better solution is get_results

Reply:
I have fixed the bug